makelib/k8s_tools.mk

```diff
@@ -97,34 +97,34 @@ $(KUBECTL):
 $(KUSTOMIZE):
 	@$(INFO) installing kustomize $(KUSTOMIZE_VERSION)
 	@mkdir -p $(TOOLS_HOST_DIR)/tmp-kustomize
-	@curl -fsSL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$(SAFEHOST_PLATFORM).tar.gz | tar -xz -C $(TOOLS_HOST_DIR)/tmp-kustomize
+	@curl -fsSL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_$(SAFEHOSTPLATFORM).tar.gz | tar -xz -C $(TOOLS_HOST_DIR)/tmp-kustomize
```

Comment:
Will this break since these two variables are not interchangeable?

Reply:
Based on https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh the kustomize install script is setting the version to "${opsys}_${arch}" -- so SAFEHOST_PLATFORM should be correct.